pkg/webhook/nodenetworkconfigurationpolicy/validation_test.go

```diff
@@ -0,0 +1,75 @@
+package nodenetworkconfigurationpolicy
+
+import (
+	"github.com/nmstate/kubernetes-nmstate/pkg/policyconditions"
```

Comment:
I think this one needs to go to the last import group

Reply:
Thanks again, [done](https://github.com/nmstate/kubernetes-nmstate/compare/80f0cadf6ded6d8004cee6f9c6bd84334feafc19..1cac7b9d5077098f535adc140e19c041557cf11c)